jest.config.js

```diff
@@ -0,0 +1,25 @@
+module.exports = {
+  roots: [
+    "<rootDir>"
+  ],
+  testPathIgnorePatterns: [
+    "/node_modules/",
+    "/dist/",
+  ],
+  collectCoverageFrom: [
+    "./src/**/*.ts"
+  ],
+  coveragePathIgnorePatterns: [
+    "/src/bin/"
+  ],
+  preset: "ts-jest",
+  testEnvironment: "node",
+  verbose: true,
+  testMatch: ["**/test/**/*.unit.[jt]s"],
+  globals: {
+    "ts-jest": {
+      diagnostics: false,
+    }
+  },
+  globalSetup: "./test/global.setup.ts"
+};
```

Comment:
jest.config.js changes made.

Can we sign off for now and we will look at these 2 when we do the screens for real rather than skeleton?Naming conventions - controllers rather than routes.
Declaring response render params within the controllers rather than using handlers.

File types used - html and ts rather than njk and js. - do we really want to be using .js,  when .ts is javascript but much better as it is typed?
there is also a node-review-web-starter-ts/basic-structure.

Reply:
The changes will need to be made to bring this project in line with the latest Companies House Standards for node.js. There are two node starters one is typescript. So I agree using the typescript version will be a good approach. There are some minor changes still to be made to the starter project so we can defer your changes until they are complete, which should only be a week or so, then going forward this code can be brought inline with the standards.